website/docs/docs/deploy/advanced-ci.md

```diff
@@ -31,6 +31,18 @@ dbt reports the comparison differences in:
 
 <Lightbox src="/img/docs/dbt-cloud/example-ci-compare-changes-tab.png" width="85%" title="Example of the Compare tab" />
 
+### Speeding up comparisons
+
+It's common for CI jobs to only [build a subset of data](/best-practices/best-practice-workflows#limit-the-data-processed-when-in-development) (for example only the last 7 days of data). 
```

Comment:
Suggest removing this since we call this out in the "scenario" bullet list below